uber/decorators.py

```diff
@@ -81,7 +81,13 @@ def _suffix_property_check(inst, name):
 def csrf_protected(func):
     @wraps(func)
     def protected(*args, csrf_token, **kwargs):
-        check_csrf(csrf_token)
+        try:
+            check_csrf(csrf_token)
+        except AssertionError as e:
```

Comment:
You say `as e` in both these cases but then don't actually print the error. If you don't plan on printing the error or using it, you can just say `except AssertionError:`